detox/src/ios/web.js

```diff
@@ -170,7 +169,7 @@ class WebElement {
   }
 
   withAction(action, traceDescription, ...params) {
-    assert(traceDescription, `must provide trace description for action: \n ${JSON.stringify(action)}`);
+    assertDefined(traceDescription);
```

Comment:
Not sure how it will work if you need to print a specific error about `traceDescription === undefined` – how do you understand that, let's say, a variable1 is `trace description`? 🤔

Reply:
done